### DISCUSSION TITLE
Unable to upgrade wiki.js

### DISCUSSION THREAD

**Discussion by MilanKrocek** [requarks/wiki#5695] [Help / Questions] [answered]
OS: Windows 10 x64 and Docker Desktop
Wiki.js version: 2.5.285
Database engine: PostgreSQL 11.16

Hello all,

I followed this guide to make wiki.js running: https://docs.requarks.io/install/docker-desktop
Today, I decided to upgrade it by this guide: https://docs.requarks.io/install/upgrade (by Docker Compose)

Once I run the second command: "docker-compose up --force-recreate". It gets stuck on the last row. Any idea what can be wrong? Thank you for your assistance.

C:\Wiki>docker-compose up --force-recreate
[+] Running 2/2
 - Container wiki-db-1    Recreated                                                                                0.3s
 - Container wiki-wiki-1  Recreated                                                                                0.3s
Attaching to wiki-db-1, wiki-wiki-1
wiki-db-1    |
wiki-db-1    | PostgreSQL Database directory appears to contain a database; Skipping initialization
wiki-db-1    |
wiki-db-1    | 2022-09-29 18:13:02.828 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
wiki-db-1    | 2022-09-29 18:13:02.828 UTC [1] LOG:  listening on IPv6 address "::", port 5432
wiki-db-1    | 2022-09-29 18:13:02.862 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
wiki-db-1    | 2022-09-29 18:13:03.026 UTC [21] LOG:  database system was shut down at 2022-09-29 18:10:08 UTC
wiki-db-1    | 2022-09-29 18:13:03.044 UTC [1] LOG:  database system is ready to accept connections
wiki-wiki-1  | Loading configuration from /wiki/config.yml... OK
wiki-wiki-1  | 2022-09-29T18:13:04.325Z [MASTER] info: =======================================
wiki-wiki-1  | 2022-09-29T18:13:04.327Z [MASTER] info: = Wiki.js 2.5.289 =====================
wiki-wiki-1  | 2022-09-29T18:13:04.327Z [MASTER] info: =======================================
wiki-wiki-1  | 2022-09-29T18:13:04.327Z [MASTER] info: Initializing...
wiki-wiki-1  | 2022-09-29T18:13:05.154Z [MASTER] info: Using database driver pg for postgres [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:05.172Z [MASTER] info: Connecting to database...
wiki-wiki-1  | 2022-09-29T18:13:05.219Z [MASTER] info: Database Connection Successful [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:05.846Z [MASTER] warn: Mail is not setup! Please set the configuration in the administration area!
wiki-wiki-1  | 2022-09-29T18:13:05.921Z [MASTER] info: Loading GraphQL Schema...
wiki-wiki-1  | 2022-09-29T18:13:06.844Z [MASTER] info: GraphQL Schema: [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:07.254Z [MASTER] info: HTTP Server on port: [ 3000 ]
wiki-wiki-1  | (node:1) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./public/" in the "exports" field module resolution of the package at /wiki/node_modules/extract-files/package.json.
wiki-wiki-1  | Update this package.json to use a subpath pattern like "./public/*".
wiki-wiki-1  | (Use `node --trace-deprecation ...` to show where the warning was created)
wiki-wiki-1  | 2022-09-29T18:13:07.274Z [MASTER] info: HTTP Server: [ RUNNING ]
wiki-wiki-1  | 2022-09-29T18:13:07.500Z [MASTER] info: No new analytics providers found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.550Z [MASTER] info: Loaded 21 authentication strategies: [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:07.603Z [MASTER] info: No new comment providers found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.649Z [MASTER] info: No new editors found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.711Z [MASTER] info: No new loggers found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.793Z [MASTER] info: No new renderers found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.824Z [MASTER] info: No new search engines found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.892Z [MASTER] info: No new storage targets found: [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.892Z [MASTER] info: Checking for installed optional extensions...
wiki-wiki-1  | 2022-09-29T18:13:07.903Z [MASTER] info: Optional extension git is installed. [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:07.907Z [MASTER] info: Optional extension pandoc was not found on this system. [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.913Z [MASTER] info: Optional extension puppeteer was not found on this system. [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.920Z [MASTER] info: Optional extension sharp was not found on this system. [ SKIPPED ]
wiki-wiki-1  | 2022-09-29T18:13:07.924Z [MASTER] info: Authentication Strategy Local: [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:08.798Z [MASTER] info: (COMMENTS/DEFAULT) Initializing...
wiki-wiki-1  | 2022-09-29T18:13:08.799Z [MASTER] info: (COMMENTS/DEFAULT) Initialization completed.
wiki-wiki-1  | 2022-09-29T18:13:08.812Z [MASTER] info: (SEARCH/POSTGRES) Initializing...
wiki-wiki-1  | 2022-09-29T18:13:08.844Z [MASTER] info: Telemetry is active: [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:08.845Z [MASTER] info: (SEARCH/POSTGRES) Initialization completed.
wiki-wiki-1  | 2022-09-29T18:13:08.850Z [MASTER] info: Purging orphaned upload files...
wiki-wiki-1  | 2022-09-29T18:13:08.851Z [MASTER] info: Syncing locales with Graph endpoint...
wiki-wiki-1  | 2022-09-29T18:13:08.852Z [MASTER] info: Fetching latest updates from Graph endpoint...
wiki-wiki-1  | 2022-09-29T18:13:08.930Z [MASTER] info: Purging orphaned upload files: [ COMPLETED ]
wiki-wiki-1  | Loading configuration from /wiki/config.yml... OK
wiki-wiki-1  | 2022-09-29T18:13:09.194Z [JOB] info: Rebuilding page tree...
wiki-wiki-1  | 2022-09-29T18:13:09.269Z [MASTER] info: Fetching latest updates from Graph endpoint: [ COMPLETED ]
wiki-wiki-1  | 2022-09-29T18:13:09.654Z [MASTER] info: Pulled latest locale updates for English from Graph endpoint: [ COMPLETED ]
wiki-wiki-1  | 2022-09-29T18:13:09.657Z [MASTER] info: Syncing locales with Graph endpoint: [ COMPLETED ]
wiki-wiki-1  | 2022-09-29T18:13:09.974Z [JOB] info: Using database driver pg for postgres [ OK ]
wiki-wiki-1  | 2022-09-29T18:13:10.065Z [JOB] info: Rebuilding page tree: [ COMPLETED ]

**Comment by MilanKrocek** [comment]
Anyone could help please? Thank you...

**Comment by NGPixel** [comment, marked as answer]
It gets "stuck" because you didn't include the `-d` flag.

```
docker-compose up --force-recreate -d
```

**Reply by MilanKrocek** [reply]
Oh, sorry for that... I completely overlooked it! Many thanks!
